pdf/pdfcanvas.cpp

```diff
@@ -113,9 +113,16 @@ PDFCanvas::PDFCanvas(QQuickItem* parent)
 
 PDFCanvas::~PDFCanvas()
 {
-    if(d->resizeTimer)
-        delete d->resizeTimer;
+    for( int i = 0; i < d->pageCount; ++i )
```

Comment:
First, this alone shouldn't do anything different than sceneGraphInvalidated(). Second, pages.count() would assume all the pages are put to QHash, which I don't think this should do.